app/controllers/spotlight/home_pages_controller.rb

```diff
@@ -10,5 +10,15 @@ module Spotlight
   # Override the upstream HomePagesController in order to inject range limit behaviors
   class HomePagesController
     include BlacklightRangeLimit::ControllerOverride
+    
+    # rubocop:disable Rails/LexicallyScopedActionFilter
+    # Tweak the authorization for the range limit actions
+    before_action :authenticate_user!, except: [:show, :range_limit, :range_limit_panel]
```

Comment:
Should we still be authenticating the user in the `show` action? Or since home pages can't be published/unpublished does the exhibit's accessibility take precedence and it doesn't really end up mattering?

Reply:
I think that's probably right. This is copied straight for exhibits where we solved the same problem.